cis_v300/section_1.fp

```diff
@@ -25,6 +25,29 @@ locals {
   }
 }
 
+locals {
+  cis_v300_1_12_pipelines = {
+    detect_and_correct_iam_users_unused_access_key    = aws_compliance.pipeline.detect_and_correct_iam_users_with_unused_access_key_45_days,
+    detect_and_correct_iam_roles_unused_login_profile = aws_compliance.pipeline.detect_and_correct_iam_users_with_unused_login_profile_45_days
+  }
+}
+
+locals {
+  cis_v300_1_16_pipelines = {
+    detect_and_correct_iam_users_with_policy_star_star  = aws_compliance.pipeline.detect_and_correct_iam_users_with_policy_star_star_attached,
+    detect_and_correct_iam_roles_with_policy_star_star  = aws_compliance.pipeline.detect_and_correct_iam_roles_with_policy_star_star_attached,
+    detect_and_correct_iam_groups_with_policy_star_star = aws_compliance.pipeline.detect_and_correct_iam_groups_with_policy_star_star_attached
+  }
+}
+
+locals {
+  cis_v300_1_22_pipelines = {
+    detect_and_correct_iam_users  = aws_compliance.pipeline.detect_and_correct_iam_users_with_unrestricted_cloudshell_full_access,
+    detect_and_correct_iam_roles  = aws_compliance.pipeline.detect_and_correct_iam_roles_with_unrestricted_cloudshell_full_access,
+    detect_and_correct_iam_groups = aws_compliance.pipeline.detect_and_correct_iam_groups_with_unrestricted_cloudshell_full_access
+  }
+}
```

Comment:
```suggestion
locals {
  cis_v300_1_12_pipelines = [
    aws_compliance.pipeline.detect_and_correct_iam_users_with_unused_access_key_45_days,
    aws_compliance.pipeline.detect_and_correct_iam_users_with_unused_login_profile_45_days
  ]

  cis_v300_1_16_pipelines = [
    aws_compliance.pipeline.detect_and_correct_iam_users_with_policy_star_star_attached,
    aws_compliance.pipeline.detect_and_correct_iam_roles_with_policy_star_star_attached,
    aws_compliance.pipeline.detect_and_correct_iam_groups_with_policy_star_star_attached
  ]

  cis_v300_1_22_pipelines = [
    aws_compliance.pipeline.detect_and_correct_iam_users_with_unrestricted_cloudshell_full_access,
    aws_compliance.pipeline.detect_and_correct_iam_roles_with_unrestricted_cloudshell_full_access,
    aws_compliance.pipeline.detect_and_correct_iam_groups_with_unrestricted_cloudshell_full_access
  ]
}
```

Could we combine into 1 local? Also, could we switch to just using lists instead of objects for each local?